src/jackett.py

```diff
@@ -183,7 +194,18 @@ def search_jackett(payload, method):
         res = jackett.search_query(payload["search_title"])
 
     log.debug(f"{method} search returned {len(res)} results")
-
+    p_dialog.update(10, message = utils.translation(32750))
     res = filter_results(method, res)
 
-    return sort_results(res)
+    res = jackett.asinc_magnet_resolve(res, p_dialog)
+
+    p_dialog.update(90, message = utils.translation(32752))
+    res = filter.unique(res)
+    log.info(f"filtering for unique items {len(res)}")
+    log.debug(f"unique items results: {res}")
+
+    p_dialog.update(95, message = utils.translation(32753))
+    res = sort_results(res)
+
+    p_dialog.update(100, message = utils.translation(32754))
+    return res[:get_setting('max_results', int)]
```

Comment:
Max results should be filtered at the API level, not retrieved, and then removed. There is no performance gain in such scenario, unless there's a different reason for setting a max results?

Reply:
if you a talking about jacket API. I would be glad to do so. But i can't find any way to limit results and keep it ordered by seed count. where is no such option in API. 
So i need to get all data from jacket in order to filter in bi seed count, and then limit.

Reply:
done